src/functionBased/components/TodosList.js

```diff
@@ -0,0 +1,19 @@
+import React from "react"
+import TodoItem from "./TodoItem"
+
+const TodosList = props => {
+  return (
+    <ul>
+      {props.todos.map(i => (
+        <TodoItem
+          key={i.id}
+          todo={i}
+          handleChangeProps={props.handleChangeProps}
+          deleteTodoProps={props.deleteTodoProps}
+          setUpdate={props.setUpdate}
```

Comment:
- I feel you can use Destructuring assignment to take value from props. to know more [click here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)